tests/contrib/gevent/test_monkeypatch.py

```diff
@@ -20,20 +20,18 @@ def test_gevent_warning(monkeypatch):
     assert b"RuntimeWarning: Loading ddtrace before using gevent monkey patching" in subp.stderr.read()
 
 
-def test_gevent_auto_patching(run_python_code_in_subprocess):
-    code = """
-import ddtrace; ddtrace.patch_all()
-
-import gevent  # Patch on import
-from ddtrace.contrib.gevent import GeventContextProvider
+@pytest.mark.subprocess
```

Comment:
This PR caused a merge conflict: https://github.com/DataDog/dd-trace-py/pull/3520/files. I'm gonna revert the changes in this file